adaptive_scheduler/monitoring/network_status.py

```diff
@@ -104,10 +104,10 @@ def has_changed(self):
         ''' Return True if the current network state is different from
             the previous network state.
         '''
-        if flatten(self.current_events) == flatten(self.previous_events):
+        _log_event_differences(self.current_events, self.previous_events)
```

Comment:
Move this logging up so we log the difference either way. This way, we should know from the logs both if there were ANY differences (from this log), and if there were resource specific differences (from the warm start disabled log later on).